client/src/components/QandA/QuestionsList/Answers.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import AnswerBody from './Answers/AnswerBody.jsx';
+import AnswerDetails from './Answers/AnswerDetails.jsx';
+import AnswerImages from './Answers/AnswerImages.jsx';
+
+const Answers = ({ answers }) => {
+  let answersArray = Object.values(answers);
+  let sortedAnswers = [];
+  let temp = [];
+
+  answersArray.sort((a, b) => b.helpfulness - a.helpfulness);
+  for (let i = 0; i < answersArray.length; i++) {
+    if (answersArray[i].answerer_name === 'Seller') {
+      sortedAnswers.push(answersArray[i]);
+    } else {
+      temp.push(answersArray[i]);
+    }
+  }
+  sortedAnswers = sortedAnswers.concat(temp);
+  temp = [];
+
+  return (
+    <>
+      {sortedAnswers.map(ans =>
+        <div>
```

Comment:
give this div a key attribute and use the ans.id for unique num

Reply:
Done.